.github/workflows/ci.yml

```
@@ -0,0 +1,56 @@
+  DOTNET_VERSION_1: '8.0.x'
+  DOTNET_VERSION_2: '9.0.x'
+  PROJECT_PATH: 'src/Aivis.Net.csproj'
+  SAMPLE_PATH: 'sample/Console/Aivis.Sample.Console.csproj'
+
+jobs:
+  build-and-test:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)